feat/add initial casefile draft for 00055 (#475)
DIVD-NL/web-csirt: _cases/2022/DIVD-2022-00055.md

---
layout: case
title: "Server Management Interfaces security issues" 
divd: DIVD-2022-00038
excerpt: "DIVD is researching vulnerabilities in (hardware) server management interfaces globally and notifying the owners of misconfigured services."
author: Pepijn van der Stap
lead: Pepijn van der Stap
status: Open
researchers:
- Tom Wolters
- Pepijn van der Stap
- Melvin Boers
cves: n/a
product: n/a
versions: any. For hardware it depends on the firmware shipped by the hardware vendor

> recommendation: "If you received a notification of a vulnerability, patch your system with the information provided in this notification."

Reviewer: ```suggestion
recommendation: "If you received a notification of a vulnerability, patch your system with the information provided in the notification."
```
If the istructions are in the notification, it should read 'the' otherwise if the instructions are on this web page, use 'this'

patch_status: n/a
start: 2022-10-08
end: 
timeline:
- start: 2022-10-08
  end:
  event: DIVD starts investigating the scope of management interfaces.
ips: n/a

---

## Summary

> DIVD started looking for vulnerabilities worldwide. This effort is aimed at vulnerabilities regarding (hardware) server management interfaces. These interfaces are often exposed to the internet and are not protected by default. This makes them a target for attackers. DIVD is researching vulnerabilities in (hardware) server management interfaces globally and notifying the owners of misconfigured services.

Reviewer: `DIVD started looking for vulnerabilities worldwide.` volgens mij is dit niet nieuw, maar deden we dit voor deze case ook al, toch?

```suggestion
DIVD started looking for  vulnerabilities regarding (hardware) server management interfaces, worldwide. These interfaces are often exposed to the internet and are not protected by default. This makes them a target for attackers. DIVD is researching vulnerabilities in (hardware) server management interfaces globally and notifying the owners of misconfigured services.
```

Author: klopt, snel geschreven, thanks!

## What you can do

* If you receive a notification, make sure the vulnerability described in that notification is patched. The notification will be sent along with a location and description of the vulnerability. If you have any questions regarding the mitigation of these vulnerabilities, feel free to reply to the email, and we'll gladly help!

> IT and IT security teams should consider putting the instance on a separate network, and only allow access to it from a trusted network. This will prevent attackers from accessing the management interface from the internet.

Reviewer: ```suggestion
IT and IT security teams should consider putting the instance on a separate network, and only allow access to it from a trusted network, via a stepping stone/bastion server, or via VPN. This will prevent attackers from accessing the management interface from the internet.
```

## What we are doing

> * DIVD is currently ensuring that the owners of vulnerable systems are being notified. We do this by scanning for vulnerable hosts, verifying the vulnerability and notifying the owners of these systems. If you receive an email from us regarding this case, the vulnerability has been confirmed.

Reviewer: Is het systeem vulnerable of heeft het een ongewenste configuratie?

Author: Kwetsbaar door de misconfiguraties (of resultaat van vendor die niet meer updated)

Reviewer: Gaan jullie testen op exposed management interfaces, of op exposed and vulnerable management services, want dat haal ik niet uit de case.

Author: . 

```suggestion
* DIVD is currently ensuring that the owners of vulnerable systems are being notified. We do this by scanning for exposed management interfaces, and then examining whether vulnerabilities are present and notifying the owners of these vulnerable systems with the issue and remediation information.
* ```

Author: Zou duidelijker moeten zijn!

## What you can do

* If you receive a notification, make sure the vulnerability described in that notification is patched. The notification will be sent along with a location and description of the vulnerability. If you have any questions regarding the mitigation of these vulnerabilities, feel free to reply to the email, and we'll gladly help!

IT and IT security teams should consider putting the instance on a separate network, and only allow access to it from a trusted network, via a stepping stone/bastion server, or via VPN. This will prevent attackers from accessing the management interface from the internet.

## What we are doing

* DIVD is currently ensuring that the owners of vulnerable systems are being notified. We do this by scanning for exposed management interfaces, and then examining whether vulnerabilities are present and notifying the owners of these vulnerable systems with the issue and remediation information.
* ```

{% include timeline.html %}

## More information

> * [Intelligent Platform Management Interface ++ Security Best Practises](http://fish2.com/ipmi/bp.pdf)

Reviewer: Is er een https link beschikbaar anders moet ik whitelisten…
```
Checking internal link hashes in 1 file
[17](https://github.com/DIVD-NL/web-csirt/actions/runs/3230377330/jobs/5288755915#step:9:18)
Ran on 255 files!
[18](https://github.com/DIVD-NL/web-csirt/actions/runs/3230377330/jobs/5288755915#step:9:19)
For the Links check, the following failures were found:
[19](https://github.com/DIVD-NL/web-csirt/actions/runs/3230377330/jobs/5288755915#step:9:20)

[20](https://github.com/DIVD-NL/web-csirt/actions/runs/3230377330/jobs/5288755915#step:9:21)
* At _site/cases/DIVD-2022-00055/index.html:564:
[21](https://github.com/DIVD-NL/web-csirt/actions/runs/3230377330/jobs/5288755915#step:9:22)

[22](https://github.com/DIVD-NL/web-csirt/actions/runs/3230377330/jobs/5288755915#step:9:23)
  http://fish2.com/ipmi/bp.pdf is not an HTTPS link
[23](https://github.com/DIVD-NL/web-csirt/actions/runs/3230377330/jobs/5288755915#step:9:24)

[24](https://github.com/DIVD-NL/web-csirt/actions/runs/3230377330/jobs/5288755915#step:9:25)

[25](https://github.com/DIVD-NL/web-csirt/actions/runs/3230377330/jobs/5288755915#step:9:26)
HTML-Proofer found 1 failure!
```
